### PR TITLE
Beacd 57

### PR DESCRIPTION
**Pull request by liampan**

### CODE REVIEW

File: app/controllers/employer/WhatEmployerTaxDoYouWantToAddController.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          BadRequest(
+            whatEmployerTaxDoYouWantToAdd(appConfig, formWithErrors, getOptions(request.request.enrolments))(
+              request.serviceInfoContent)),
+        (value) => Redirect(navigator.nextPage(WhatEmployerTaxDoYouWantToAddId, (value, request.request.enrolments)))
+      )
+  }
+
+  def getOptions(enrolments: Enrolments): Seq[RadioOption] = {
+    val hasEPAYE: Boolean = utils.Enrolments.hasEnrolments(enrolments, HmrcEnrolmentType.EPAYE)
+    if (hasEPAYE) options.filterNot(_ == RadioOption("whatEmployerTaxDoYouWantToAdd", EPAYE.toString))
```
Comment (kurtlogan) [inline app/controllers/employer/WhatEmployerTaxDoYouWantToAddController.scala]: Use `_.value == EPAYE.toString` instead. It is a little clearer and doesn't depend on a string value that could potentially change

File: conf/application.conf [hunk 2]
```diff
@@ -213,8 +213,8 @@ urls {
       businessRegistration = "/business-registration/introduction"
       mossRegistration = "/moss-registration/org/introduction"
       vatmoss = "/moss-registration/org/provide-vat-details"
+      enrolERS =  "/ers/org///add-scheme"
```
Comment (kurtlogan) [inline conf/application.conf]: Why so many `/`?

**Comment by kurtlogan** [comment]
2 minor comments

File: conf/application.conf (hunk 2 above)
Comment (liampan) [reply conf/application.conf]: is as is

**Pull request merged**